pkg/cloud/graph.go

```diff
@@ -120,19 +120,19 @@ func (c *AzureClient) GetApplication(ctx context.Context, displayName string) (*
 
 // DeleteServicePrincipal deletes a service principal.
 func (c *AzureClient) DeleteServicePrincipal(ctx context.Context, objectID string) error {
-	log.Debugf("Deleting service principal with object id=%s", objectID)
+	mlog.Debug("Deleting service principal", "object id", objectID)
```

Comment:
ditto comment for all keys that have 2 words (`should we use camel case key name?`)